packages/loaders/src/Progress.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const COMPONENT_ID = 'loaders.progress';
+
+const { zdColorGreen600, zdColorGrey200, zdSpacingXs } = require('@zendeskgarden/css-variables');
+
+const heights = {
+  small: 2,
+  medium: 6,
+  large: 12
+};
+
+const sizeToBorderRadius = size => heights[size] / 2;
+
+const ProgressBackground = styled.div.attrs(props => ({
+  borderRadius: sizeToBorderRadius(props.size)
+}))`
+  margin: ${zdSpacingXs} 0;
+  border-radius: ${({ borderRadius }) => borderRadius}px;
+  background-color: ${zdColorGrey200};
+  color: ${({ color }) => color || zdColorGreen600};
+`;
+
+const ProgressIndicator = styled.div.attrs(props => ({
+  height: heights[props.size],
+  borderRadius: sizeToBorderRadius(props.size)
+}))`
+  transition: width 0.1s ease-in-out;
+  border-radius: ${({ borderRadius }) => borderRadius}px;
+  background: currentColor;
+  width: ${({ value }) => value}%;
+  height: ${({ height }) => height}px;
+`;
+
+const Progress = ({ value, size, ...other }) => {
```

Comment:
Should we be using `React.forwardRef` here? This would allow our Tooltip and other positioning packages to position PopperJS properly with `ref`.